rocksdb_admin/helix_client.cpp

```diff
@@ -117,10 +117,10 @@ void invokeClass(JNIEnv* env,
 
   // The participant main function should never exit
   env->ExceptionDescribe();
-  CHECK(false) << "Participant main() exited";
```

Comment:
Would it be simpler to call  helixManager.disconnect() in ShutdownJVM() directly?
We can keep the CHECK() here and let it fail loudly if the participant fails to join helix cluster on startup.

Reply:
Great suggestion, thanks!